core/pubkeyConverter/bech32PubkeyConverter_test.go

```diff
@@ -1,29 +1,29 @@
 package pubkeyConverter_test
 
 import (
+	"encoding/hex"
 	"errors"
 	"fmt"
 	"strings"
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
-	"github.com/ElrondNetwork/elrond-go-core/core/mock"
 	"github.com/ElrondNetwork/elrond-go-core/core/pubkeyConverter"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestNewBech32PubkeyConverter_InvalidSizeShouldErr(t *testing.T) {
 	t.Parallel()
 
-	bpc, err := pubkeyConverter.NewBech32PubkeyConverter(-1, &mock.LoggerMock{})
+	bpc, err := pubkeyConverter.NewBech32PubkeyConverter(-1, "erd")
```

Comment:
all these "erd" could have been declared as a const

Reply:
const in `core/constants` added.